src/usart.rs

```diff
@@ -447,6 +447,27 @@ impl Peripheral for pac::USART2 {
     type Tx = swm::U2_TXD;
 }
 
+#[cfg(feature = "845")]
+impl Peripheral for pac::USART3 {
+    /// USART3 and PIN_INT6 share an interrupt, this may cause difficulties
+    /// when trying to use both at the same time
+    // Since we don't provide an abstraction for pin interrupts,
+    //
+    const INTERRUPT: Interrupt = Interrupt::PIN_INT6_USART3;
+
+    type Rx = swm::U3_RXD;
+    type Tx = swm::U3_TXD;
+}
+
+#[cfg(feature = "845")]
+impl Peripheral for pac::USART4 {
+    /// USART4 and PIN_INT7 share an interrupt, this may cause difficulties
+    /// when trying to use both at the same time
```

Comment:
Same of above, might be worth putting that into the documentation of the respective field in `Peripherals`.